core/templates/dev/head/domain/feedback_thread/FeedbackThreadSummaryObjectFactory.ts

```diff
@@ -17,13 +17,29 @@
    domain objects.
  */
 
-var oppia = require('AppInit.ts').module;
+import { Injectable } from '@angular/core';
+import { downgradeInjectable } from '@angular/upgrade/static';
+
+export class FeedbackThreadSummary {
+  status: string;
+  originalAuthorId: string;
+  lastUpdated: Date;
```

Comment:
In some places in this PR, lastUpdated is a Date, in others it is a number. This may be correct but I just wanted to check that you've confirmed it is indeed correct?

Reply:
I saw the type from the [current develop](https://github.com/oppia/oppia/blob/d89d68f3b682942c0d43a8a368c52bf815d054b5/core/templates/dev/head/domain/feedback_thread/FeedbackThreadSummaryObjectFactory.ts#L51) and left it unchanged.